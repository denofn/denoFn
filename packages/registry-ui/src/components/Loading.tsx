import React from "react";
import { css } from "otion";

import { CenterAlign } from "./CenterAlign";
import { Spinner } from "./Spinner";

export function Loading() {
  return (
    <CenterAlign>
      <Spinner />
      <h3
        className={css({
          display: "inline-block",
          lineHeight: "2rem",
          margin: "0",
          marginLeft: ".5rem",
          alignSelf: "top",
        })}
      >
        Loading...
      </h3>
    </CenterAlign>
  );
}
