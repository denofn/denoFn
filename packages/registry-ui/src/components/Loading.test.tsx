import { render, cleanup } from "@testing-library/react";
import React from "react";

import { Loading } from "./Loading";

afterEach(cleanup);

it("renders with the correct styles", () => {
  const { asFragment } = render(<Loading />);
  expect(asFragment()).toMatchSnapshot();
});
