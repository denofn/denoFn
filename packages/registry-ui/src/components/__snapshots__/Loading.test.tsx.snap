// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with the correct styles 1`] = `
<DocumentFragment>
  <div
    class="_1d3w5wq _1o3nkn _1tx1l5v _zjik7 _1h3rtzg _f7ay7b"
  >
    <div
      class="_1r5gb7q _14ztwpi _pmoan0 _eqx0xi _ye6num _11x1ul3 _fxc8cu _1i432rq _1acr3ak _j033lg _13nzo53"
    />
    <h3
      class="_1r5gb7q _1t5azn2 _ti75j2 _1qphwdi _f7j4u9"
    >
      Loading...
    </h3>
  </div>
</DocumentFragment>
`;
